### PR TITLE
better UI for viewing submissions that are involved in edits

### PR DESCRIPTION
**Pull request by czue**
changes like the below
image

image

### CODE REVIEW

**Comment by czue** [comment]
@dannyroberts @TylerSheffels

File: corehq/apps/reports/templatetags/xform_tags.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from functools import partial
+from django.contrib.humanize.templatetags.humanize import naturaltime
```
Comment (dimagimon) [inline corehq/apps/reports/templatetags/xform_tags.py]: F401 'naturaltime' imported but unused

File: corehq/apps/reports/templates/reports/form/partials/single_form.html [hunk 2]
```diff
@@ -174,7 +191,7 @@
             {% endif %}
             {% if user.can_edit_data %}
             <p>
-                {% if not is_archived %}
+                {% if not is_archived and not edit_info.was_edited %}
```
Comment (TylerSheffels) [inline corehq/apps/reports/templates/reports/form/partials/single_form.html]: This looks like it prevents a user from archiving an edited form, is there a reason for that?

**Comment by TylerSheffels** [comment]
One question, otherwise good w/ this.

File: corehq/apps/reports/templates/reports/form/partials/single_form.html (hunk 2 above)
Comment (czue) [inline corehq/apps/reports/templates/reports/form/partials/single_form.html]: an edited form already doesn't contribute to the case or show up in exports so there shouldn't need to be a reason to archive it.

also, if you unarchive an archived form it was going to be a bit more annoying to check whether it was an edit or not and then properly set the `doc_type` - so it was a bit of me being lazy

**Pull request merged**